cmd/podman/common.go

```diff
@@ -388,6 +388,10 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"network", getDefaultNetwork(),
 		"Connect a container to a network",
 	)
+	createFlags.Bool(
+		"no-hosts", false,
```

Comment:
Why are you differentiating between /etc/hosts and resolv.conf?

Reply:
I can see cases where you want to manage only one or the other, and I don't think it makes sense for `--dns=none` to disable both